jest.config.js

```diff
@@ -1,6 +1,7 @@
 const esModules = ['react-stylable-diff'].join('|');
 
 module.exports = {
+	preset: 'ts-jest/presets/js-with-ts',
```

Comment:
**Important change** — we're using the `js-with-ts` preset to support interop between TS and ESM modules when running tests (since many utils are still written in JS).